### PR TITLE
[plugin.video.tvvn] v0.9.11

### PR DESCRIPTION
**Pull request by zecoj**
version 0.9.11 (19/09/2017):
 * fix translatePath as per http://kodi.wiki/view/Add-on_rules#Requirements_for_scripts_and_plugins

version 0.9.10 (19/09/2017):
 * remove all broken sources, TVNet (VTC) is the only active one.
 * compressed all images properly

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by zecoj** [comment]
@learningit @romanvm  Can you guys please have a look at this? The current version on the repo is broken and users are complaining... FYI this plugin has been on the repo since mailing list submission was used but this is the first commit on here.

File: plugin.video.tvvn/tvvn.py [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+        if (mysettings.getSetting('descriptions')=='true' and desc != ''):
+                if mysettings.getSetting('descriptions_on_right') == 'false':
+                        name = desc+"    "+name
+                else:
+                        name = name+"    "+desc
+
+        give_url = sys.argv[0]+"?mode=1&chn="+namex+"&src="+src
+        liz = xbmcgui.ListItem( name, iconImage=xbmc.translatePath(os.path.join(home, 'resources', 'logos', iconimage)), thumbnailImage=xbmc.translatePath(os.path.join(home, 'resources', 'logos', iconimage)))
+        liz.setInfo(type="Video", infoLabels={"Title": name})
+        liz.setProperty("Fanart_Image",fanart)
```
[the remark this answers is not in the document]
Comment (romanvm) [reply plugin.video.tvvn/tvvn.py]: FYI: https://codedocs.xyz/xbmc/xbmc/group__python__xbmcgui__listitem.html#gad3f9b9befa5f3d2f4683f9957264dbbe

**Comment by zecoj** [comment]
Thanks @romanvm I've updated as per suggested and recommitted.

**Comment by romanvm** [comment]
All changes should be in one commit. Please look at some online tutorials on how to rebase/squash several commits into one.

**Comment by zecoj** [comment]
Thanks @romanvm. Squashed.

**Comment by romanvm** [comment]
LGTM

**Comment by MartijnKaijser** [comment]
@zecoj can you change thee following in next update?

PROBLEM: Found strings.xml in folder /home/travis/build/xbmc/repo-plugins/plugin.video.tvvn please migrate to strings.po.
WARNING: Using the old language directory structure, please move to the new one.

See:
http://kodi.wiki/view/language_support#What_is_strings.po

**Comment by zecoj** [comment]
@MartijnKaijser sure mate. Change's made and will be included in the next update.

**Pull request merged**